### PR TITLE
some tweaks to the directory route loader

### PR DESCRIPTION
**Pull request by xabbuh**
Make some tweaks suggested by @Ener-Getick in #1339.

### CODE REVIEW

**Comment by Ener-Getick** [comment]
Ah right, I completely forgot this :+1:

File: Routing/Loader/RestRouteLoader.php [hunk 1]
```diff
@@ -162,7 +162,7 @@ private function getControllerLocator($controller)
      */
     protected function findClass($file)
     {
-        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.8 and will be removed in 2.0. Use '.__NAMESPACE__.'\ClassUtils::findClassInFile() instead.', E_USER_DEPRECATED);
+        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.8 and will be removed in 2.0.', E_USER_DEPRECATED);
```
Comment (Ener-Getick) [inline Routing/Loader/RestRouteLoader.php]: What about using ``sprintf`` here?

Comment (xabbuh) [inline Routing/Loader/RestRouteLoader.php]: I don't have any preference here. If you favour `sprintf()`, I can change it.

Comment (Ener-Getick) [inline Routing/Loader/RestRouteLoader.php]: It is in the [symfony best practices](http://symfony.com/doc/current/contributing/code/standards.html#structure) so I think it is better to use it.

Comment (xabbuh) [inline Routing/Loader/RestRouteLoader.php]: yeah, changed it :)

**Comment by Ener-Getick** [comment]
Status: Reviewed
:+1:

**Pull request merged**